addon/mixins/route.js

```diff
@@ -168,11 +168,13 @@ export default Ember.Mixin.create({
         this.set('_currentPage', startingPage);
         this.set('_totalPages', totalPages);
         infinityModel.set('reachedInfinity', !this.get('_canLoadMore'));
-        Ember.run.scheduleOnce('afterRender', this, 'infinityModelUpdated', {
-          lastPageLoaded: startingPage,
-          totalPages: totalPages,
-          newObjects: infinityModel
-        });
+        if(this.infinityModelUpdated) {
```

Comment:
Just checked again, these changes are the only changes needed in order to get the ember-release (with ember-data > 2.0.0 as supported) build to succeed.